### ISSUE TITLE
failure to create attachments sub directory

### ISSUE THREAD

**Issue by rchen19** [tagatac/bagoup#30]
**Describe the bug**
When running the program with `-a` flag to copy attachments, saw Error that says the directory already exists

**To Reproduce**
1. Running in terminal `bagoup -o /Users/***/Downloads/messages_export -c /Users/***/Downloads/all_contacts.vcf --separate-chats -a`
2. See error: 

```
2022/09/11 20:24:07 BAGOUP RESULTS:
bagoup version: 2.1.1 Darwin/arm64
Export folder: "/Users/***/Downloads/messages_export"
Export files written: 14
Chats exported: 14
Messages exported: 25
Attachments copied: 0
Attachments referenced or embedded: 0
Attachments embedded: 0
Attachments missing (see warnings above): 0
HEIC conversions completed: 0
HEIC conversions failed (see warnings above): 0
Time elapsed: 2.145910917s
2022/09/11 20:24:07 ERROR: export chats: create directory "/Users/***/Downloads/messages_export/+1**********/attachments": mkdir /Users/***/Downloads/messages_export/+1**********/attachments: file exists
```

**Expected behavior**
If the existence of a directory prevents mkdir, it should be skipped

**Screenshots**

**Environment (please complete the following information):**
 - OS: macOS 12.5.1
 - Version bagoup version 2.1.1 Darwin/arm64

**Comment by tagatac** [comment]
Thanks for the report! This should be fixed on the main branch. Want to try it out before I cut a release?

**Comment by rchen19** [comment]
> Thanks for the report! This should be fixed on the main branch. Want to try it out before I cut a release?

Thank you, that would be great!

**Comment by tagatac** [comment]
Just let me know when you've had a chance to try it.

**Comment by tagatac** [comment]
@rchen19 this fix is available via homebrew now (v2.1.2). Thanks again for your report 🙌

**Comment by rchen19** [comment]
> @rchen19 this fix is available via homebrew now (v2.1.2). Thanks again for your report 🙌

Now it takes a little over 1 min to export 30GB of content, great job!

```
2022/09/17 11:53:51 BAGOUP RESULTS:
bagoup version: 2.1.2 Darwin/arm64
Export folder: "/Users/***/Downloads/messages_export"
Export files written: 1383
Chats exported: 1383
Messages exported: 402003
Attachments copied: 38696
	application/octet-stream: 12114
	image/png: 3850
	image/tiff: 79
	image/gif: 228
	video/mp4: 145
	text/vcard: 7
	image/jpeg: 16847
	application/vnd.apple.pkpass: 22
	application/x-pkcs12: 3
	application/pdf: 20
	application/epub+zip: 1
	text/plain: 5
	image/heic: 4771
	audio/amr: 63
	video/quicktime: 466
	text/x-vlocation: 50
	image/webp: 22
	text/rtf: 1
	audio/x-m4a: 2
Attachments referenced or embedded: 40766
	application/pdf: 63
	video/mp4: 158
	application/x-pkcs12: 3
	image/jpeg: 17592
	image/heic: 4883
	image/gif: 232
	image/png: 4177
	video/quicktime: 481
	image/webp: 23
	text/vcard: 9
	text/rtf: 1
	application/vnd.apple.pkpass: 61
	application/epub+zip: 1
	audio/x-m4a: 4
	audio/mpeg: 1
	application/octet-stream: 12849
	audio/amr: 80
	text/x-vlocation: 53
	image/tiff: 90
	text/plain: 5
Attachments embedded: 0
Attachments missing (see warnings above): 2070
HEIC conversions completed: 0
HEIC conversions failed (see warnings above): 0
Time elapsed: 1m13.412186083s
```
